controller/controller.js

```diff
@@ -11,4 +11,16 @@ const sendTopics = function (req, res, next) {
 const sendEndpoints = function (req, res, next) {
   res.status(200).send(endpoints);
 };
-module.exports = { sendTopics, sendEndpoints };
+
+const sendArticleId = function (req, res, next) {
+  const articleId = Number(req.params.article_id);
+  getArticleId(articleId)
+    .then((article) => {
+      res.status(200).send(article);
+    })
```

Comment:
You could refactor this to `const {articleId} = req.params` and return the article restructured. Let me know if you need help with this 👍

Reply:
oops didn't see this will update this now